### PR TITLE
Extract email utility script

### PR DESCRIPTION
**Pull request by dorsha**

### CODE REVIEW

File: Scripts/script-ExtractEmail.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  var found;
+
+  while (found = reg.exec(text)) {
+   if (found && whitelist.indexOf(found[0]) < 0) {
+       matches[found[0]] = true;
+   }
+  }
+
+  var uniqueMatches = Object.keys(matches);
+  var emailContext = { Email: [] };
```
Comment (slavikm) [inline Scripts/script-ExtractEmail.yml]: According to our taxonomy, it should go under `Account.Email` and not `Email.Data`

File: Scripts/script-ExtractEmail.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+comment: Extract Emails from the given text and place them both as output and in the
+  context of a playbook. If given an object, will convert to JSON.
+enabled: true
+args:
+- name: text
+  required: true
+  default: true
+  description: The text to extract Emails from. If object will convert to JSON.
+outputs:
+- contextPath: URL.Data
```
Comment (slavikm) [inline Scripts/script-ExtractEmail.yml]: Copy-paste?

Comment (dorsha) [inline Scripts/script-ExtractEmail.yml]: Fixed

File: Scripts/script-ExtractEmail.yml (hunk 1 above)
Comment (aviadl) [inline Scripts/script-ExtractEmail.yml]: Please look at pipl integration
we have email under person

**Comment by dorsha** [comment]
Pushed, please review again.

**Pull request merged**